packages/flutter/lib/src/widgets/nested_scroll_view.dart

```diff
@@ -686,6 +686,11 @@ class _NestedScrollCoordinator implements ScrollActivityDelegate, ScrollHoldCont
     );
   }
 
+  @override
+  Simulation? updateBallisticAnimation(double initVelocity, double initPosition) {
```

Comment:
If this is not going to be implemented in the NestedScrollView, there should be an issue filed and a comment added here that references it.

Reply:
I have added documentation.